pkg/server/container_create.go

```diff
@@ -192,8 +192,9 @@ func (c *criService) CreateContainer(ctx context.Context, r *runtime.CreateConta
 	meta.ImageRef = image.ID
 	meta.StopSignal = image.ImageSpec.Config.StopSignal
 
-	// Get container log path.
-	if config.GetLogPath() != "" {
+	// Validate log paths and compose full container log path.
+	logrus.Debugf("Composing container full log path using sandbox log dir %q and container log path %q", sandboxConfig.GetLogDirectory(), config.GetLogPath())
```

Comment:
was more thinking that we needed log output for the else or ```if sandboxConfig.GetLogDirectory() == "" || config.GetLogPath() == ""``` case when we are not composing the full log path

Reply:
Added more verbose logging in the `else` branch as well.